docs/content/en/docs/workflows/debug.md

```diff
@@ -237,7 +239,14 @@ a debug launcher that examines the app command-line.
 
 Furthermore, your app must be built with the `--configuration Debug` options to disable optimizations.
 
-**Note for users of [VS Code's debug adapter for C#](https://github.com/OmniSharp/omnisharp-vscode):**
+
+{{< alert title="JetBrains Rider" >}}
+This set up does not yet work automatically with Cloud Code for IntelliJ in JetBrains Rider,
+but [there is a workaround](https://github.com/GoogleCloudPlatform/cloud-code-intellij/wiki/Manual-set-up-for-remote-debugging-in-Rider).
```

Comment:
nit: but maybe "there is a **manual** workaround"? to emphasis the distinction between a possible future automated solution